app-cat-clicker/Controllers/cat3controller.js

```diff
@@ -0,0 +1,50 @@
+//need to refractor h-w-1 section 3
+(function(module) {
+  module.controller("cat3Controller", cat3Controller);
+
+})(angular.module("catclick"));
+
+function cat3Controller($scope) {
```

Comment:
$scope is not being used in the function, inject dependencies that are being used in the controller function